src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -565,6 +566,40 @@ public void requestTeamReviewers(List<GHTeam> teams) throws IOException {
                 .send();
     }
 
+    /**
+     * Set the base branch on the pull request
+     *
+     * @param newBaseBranch
+     *            the name of the new base branch
+     * @throws IOException
+     *             the io exception
+     */
+    public void setBaseBranch(String newBaseBranch) throws IOException {
+        owner.root.getClient()
+                .sendRequest(
+                        root.createRequest().method("PATCH").with("base", newBaseBranch).withUrlPath(getApiRoute()),
+                        null);
+    }
+
+    /**
+     * Updates the branch. The same as pressing the button in the web GUI.
+     *
+     * @throws IOException
+     *             the io exception
+     */
+    @Preview
+    @Deprecated
+    public void updateBranch() throws IOException {
+        owner.root.getClient()
+                .sendRequest(
+                        root.createRequest()
+                                .withPreview(LYDIAN)
+                                .method("PUT")
+                                .with("expected_head_sha", head.getSha())
+                                .withUrlPath(getApiRoute() + "/update-branch"),
+                        null);
```

Comment:
Use `fetchHttpStatusCode()`.  Maybe need to verify value is `202` and throw if not.  Not sure.